packaging/fedora/myproxy.spec

```diff
@@ -387,6 +387,9 @@ fi
 %doc %{_pkgdocdir}/LICENSE*
 
 %changelog
+* Wed Dec 23 2020 Mattias Ellert <mattias.ellert@physics.uu.se> - 6.2.7-1
+- Update default run directory from /var/run to /run
```

Comment:
But during runtime this new version behaves differently than the current version. What else should we mention here then?

Reply:
Not sure, but normally we mention only packaging changes in the spec changelog, at least that's what I normally do. Otherwise I would put something like "bump version" or so (I can't really find a proper reference in the EPEL and Fedora packaging guidelines other than https://docs.pagure.org/docs-fedora/rpm-packaging-basics.html).

Reply:
Aren't there any Fedora/EPEL packages that had to change that - i.e. `/var/run` => `/run` - too, in the past? We could check the changelogs there for any guidance.

****

In general I don't have a strong opinion here, but if we did some thing like that in the past, I don't think we need to take it too serious this time.

Reply:
@msalle, @ellert 
Ok, I found https://metadata.ftp-master.debian.org/changelogs//main/o/openssh/openssh_7.9p1-10+deb10u2_changelog where it reads:

```
openssh (1:7.4p1-10) unstable; urgency=medium

  * Move privilege separation directory and PID file from /var/run/ to /run/
    (closes: #760422, #856825).
  * Unbreak Unix domain socket forwarding for root (closes: #858252).

 -- Colin Watson <cjwatson@debian.org>  Thu, 30 Mar 2017 11:19:04 +0100
```

Checking [the "version before" package](http://snapshot.debian.org/archive/debian/20170316T211946Z/pool/main/o/openssh/openssh-server_7.4p1-9_amd64.deb) and the [package corresponding to the above changelog entry](http://snapshot.debian.org/archive/debian/20170330T151744Z/pool/main/o/openssh/openssh-server_7.4p1-10_amd64.deb), there are references to `/var/run` (for 7.4p1-9) that changed to `/run` (for 7.4p1-10) included in e.g. the `control.tar.gz/postinst` files. Sure, this constitutes a change in packaging for the DEB package, but only because Debian includes service configuration into their packaging files. For our RPM the change is included in the new source tarball (inside the systemd service/unit/whatsoever file).

Considering that I'd say, let's merge this as is.

So I'm going merge this at around 18:00h CET if you don't object until then.